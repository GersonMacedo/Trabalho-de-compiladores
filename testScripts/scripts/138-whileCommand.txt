! -s SYNTACTIC
! should accept while command
program p ;
begin
    while true do
        a := a + 1
    ;
end .