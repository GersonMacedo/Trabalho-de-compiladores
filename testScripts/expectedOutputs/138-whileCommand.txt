
Performing up to syntactic analysis at testScripts/scripts/138-whileCommand.txt
No errors found
